torch/csrc/jit/ir/ir.h

```diff
@@ -251,6 +251,7 @@ struct Value {
   }
 
   TORCH_API void replaceFirstUseWith(Value* newValue);
+  TORCH_API void replaceLastUseWith(Value* newValue);
```

Comment:
Why this and firstUse needs to be a `TORCH_API`? I.e. it it supposed to be called from the external library (correct me if I'm wrong, but both torch/csrc/jit/ir/ir.cpp and torch/csrc/jit/passes/onnx/fixup_onnx_controlflow.cpp are compiled as part of `libtorch_cpu`?)

The reason I'm asking this: Windows have hardcoded max of 64K exported entires and TorchCPU exposes 60+k of them right now.

Reply:
A revisit on replacing nodes API that we already had, I think in this case, we can utilize on an exisiting one. For TORCH_API, taking out TORCH_API from `replaceFirstUseWith` breaks libtorch. I guess cause `replaceFirstUseWith` is legacy and used in `replaceAllUseWith` which is heavily used across whole Pytorch repo, so I will remain that one.